Support/bone101/UI/bbui.js

```diff
@@ -921,8 +1065,8 @@ var UI = (function() {
                 coords[0] = event.clientX + document.body.scrollLeft + document.documentElement.scrollLeft;
                 coords[1] = event.clientY + document.body.scrollTop + document.documentElement.scrollTop;
             }
-            //console.log("Position = " + [x, y]);
-            //console.log("Offset = " + [rect.left, rect.top]);
+            // console.log("Position = " + [x, y]);
```

Comment:
Please don't make whitespace changes in the same patches as other changes. Fix your text editor.

Also, to have a standard, read the README.md!

To simplify style conflicts, please use the following tools to clean-up the styles:

JS/CSS/HTML: https://www.npmjs.com/package/js-beautify version 1.5.10